cms/templates/partials/nhs_components/search_content_type.html

```diff
@@ -1,58 +1,79 @@
+    </h2>
+  </legend>
+  <div class="nhsuk-radios nhsuk-radios--conditional show-checkbox">
+    <div class="nhsuk-radios__item">
+      <input class="nhsuk-radios__input" id="content_type-1"  name="content_type" type="radio" value="pages" aria-expanded="false" {% if content_type == "pages" %}checked="true"{% endif %}>
+      <label class="nhsuk-label nhsuk-radios__label" for="content_type-1">
+        Webpage
+      </label>
+    </div>
+    <div class="nhsuk-radios__item">
+      <input class="nhsuk-radios__input" id="content_type-2" name="content_type" type="radio" value="news" aria-expanded="false" {% if content_type == "news" %}checked="true"{% endif %}>
+      <label class="nhsuk-label nhsuk-radios__label" for="content_type-2">
+        News
+      </label>
+    </div>
+    <div class="nhsuk-radios__item">
+      <input class="nhsuk-radios__input" id="content_type-3" name="content_type" type="radio" value="blogs" aria-expanded="true" {% if content_type == "blogs" %}checked="true"{% endif %}>
+      <label class="nhsuk-label nhsuk-radios__label" for="content_type-3">
+        Blogs
+      </label>
+    </div>
+    <div class="nhsuk-radios__item">
+      <input class="nhsuk-radios__input" id="content_type-4" name="content_type" type="radio" value="publications" aria-controls="conditional-publications" aria-expanded="true" {% if content_type == "publications" %}checked="true"{% endif %}>
+      <label class="nhsuk-label nhsuk-radios__label" for="content_type-4">
+        Publication
+      </label>
+    </div>
+  </div>
+</fieldset>
+<div class="nhsuk-radios nhsuk-radios--conditional show-checkbox nhsuk-u-margin-top-4">
+  {% if content_type == "publications" %}
+    <div class="nhsuk-radios__conditional" id="conditional-publications">
+  {% else %}
+    <div class="nhsuk-radios__conditional nhsuk-radios__conditional--hidden" id="conditional-publications">
+  {% endif %}
+    <h2 class="nhsuk-heading-xs">Publication type:</h2><div class="nhsuk-filter-clear " id="search-filter-hint">
+      <a href="#">Clear filters</a>
+    </div>
+    <div class="nhsuk-filter-selected" id="search-filter-hint">
+      1 Selected
+    </div>
     <div class="nhsuk-form-group">
-      <div class="nhsuk-radios">
-        <div class="nhsuk-radios__item">
-          <input class="nhsuk-radios__input" id="ct1" name="content_type" type="radio"
-          value="pages"{% if content_type == 'pages' %}checked{% endif %}>
-          <label class="nhsuk-label nhsuk-radios__label" for="ct1">Web page</label>
+      <div class="nhsuk-checkboxes" data-module="nhsuk-checkboxes">
+        <div class="nhsuk-checkboxes__item">
+          <input class="nhsuk-checkboxes__input" id="search-filter-4" name="search-filter" type="checkbox" value="farm">
+          <label class="nhsuk-label nhsuk-checkboxes__label" for="search-filter-4">
+            Household goods
```

Comment:
If this is just filler text because publication type filters aren't hooked up yet, can we wrap in a `{% comment %}` block so it never makes it into the rendered output?